vignettes/bcmaps.Rmd.orig

```diff
@@ -0,0 +1,182 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+-->
+
+
+```{r options, include=FALSE}
+knitr::opts_chunk$set(collapse = TRUE,
+  comment = "#>",
+  warning = FALSE,
+  message = FALSE,
+  fig.path = "vignette-fig-",
+  out.width = "100%",
+  fig.width = 7,
+  fig.height = 4,
+  dev = "CairoPNG",
+  dpi = 400)
+```
+
+
+## Overview
+
+`bcmaps` is an [R](https://www.r-project.org) package of spatial map layers for British Columbia.
+
+
+`bcmaps` provides access to various spatial layers of British Columbia, such as
+administrative boundaries, natural resource management boundaries, watercourses, census boundaries, etc. All layers are available as `sf` objects in the
+[BC Albers](https://spatialreference.org/ref/epsg/nad83-bc-albers/) projection, which is the B.C. Government standard.
+
+Most layers are accessed directly from the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/) using the [bcdata](https://github.com/bcgov/bcdata) R package under the hood. See each layers individual help file for more detail.
+
+
+> _**IMPORTANT NOTE** Support for Spatial objects (`sp`) is deprecated in {bcmaps} v1.2.0, and will be removed in Summer 2023. Please use `sf` objects with {bcmaps}._ A discussion on the evolution of the spatial software stack in R can be found here: https://r-spatial.org/r/2022/04/12/evolution.html.
+
+
+
+## Usage
+
+To see the layers that are available, run the `available_layers()` function:
+
+```{r, echo=FALSE, warning=FALSE}
+library(bcmaps)
+```
+
+```{r, eval=FALSE}
+library(bcmaps)
+available_layers()
+```
+
+Most layers are accessible by a shortcut function by the same name as the object.
+Then you can use the data as you would any `sf` object. The first time
+you try to access a layer, you will be prompted for permission to download that layer
+to your hard drive. Subsequently that layer is available locally for easy future access. For example:
+
+```{r}
+library(sf)
+
+bc <- bc_bound()
+plot(st_geometry(bc))
+```
+
+There are two British Columbia boundary map layers available in `bcmaps`:
+
+- `bc_bound()` for fast plotting with suitable detail for most figures (source: [(7.5M) Provinces and States - The Atlas of Canada Base Maps for BC](https://catalogue.data.gov.bc.ca/dataset/b9bd93e1-0226-4351-b943-05c6f80bd5da))
+- `bc_bound_hres()` for geospatial analyses requiring detailed lineworks of British Columbia (source: [Province of British Columbia - Boundary Terrestrial](https://catalogue.data.gov.bc.ca/dataset/30aeb5c1-4285-46c8-b60b-15b1a6f4258b))
+
+### Simple Features objects
+
+By default, all layers are returned as [`sf` spatial objects](https://cran.r-project.org/package=sf):
+
+```{r plot-maps, fig.height=5, warning=FALSE}
+library(bcmaps)
+library(sf)
+
+# Load and plot the boundaries of B.C.
+
+bc <- bc_bound()
+plot(st_geometry(bc))
```

Comment:
Going off memory, I think maybe a `reset = FALSE` here will help the plotting issue you were talking about? But if working well enough I say leave it.

Reply:
That did it!  🙌